tests/D2L.CodeStyle.Analyzers.Test/Async/Generator/AsyncToSyncMethodTransformerTests.cs

```diff
@@ -120,6 +120,33 @@ internal sealed class AsyncToSyncMethodTransformerTests {
 		Assert.AreEqual( "[Blocking] void Bar() { Foo fooFirst = m_foo.Bar( barrer ); }", actual.Value.ToFullString() );
 	}
 
+	[Test]
+	public void CompletedTaskConfigureAwait() {
+		var actual = Transform( @"[GenerateSync] async Task BarAsync() { Task.CompletedTask.ConfigureAwait(false); }" );
+
+		Assert.IsTrue( actual.Success );
+		Assert.IsEmpty( actual.Diagnostics );
+		Assert.AreEqual( "[Blocking] void Bar() { ; }", actual.Value.ToFullString() );
+	}
+
+	[Test]
+	public void FromResult() {
+		var actual = Transform( @"[GenerateSync] async Task BarAsync() { return Task.FromResult( baz ); }" );

```

Comment:
Hm, this one is `async Task` but is returning a `Task<T>`... Not sure exactly what you wanted to test here, but I think you wanted the return type to be `Task<T>` and not to have this be `async`?